src/components/EnketoForm.tsx

```diff
@@ -34,13 +37,16 @@ export const EnketoForm: React.FC<EnketoFormProps> = ({ formUID, formODKXML, ins
                 }
             })
             .catch((error) => {
-                log.error('Error adding form draft to local database:');
-                log.error(error);
+                // log.error('Error adding form draft to local database:');
+                // log.error(error);
+                console.log(error);
```

Comment:
You should be using `log.` not `console.log()` - the whole system is set up with winston so you get not only the console logs but also a log file... if you just want to change what is visible then change the loglevel in log.tsx?

Reply:
If it's an XML object you should just be able to use log.info(XML) without JSON.stringify. That works for me for HTML and XML docunents.

Reply:
Wait. Sorry. You need to use a serializer, e.g. `log.info(new XMLSerializer(XML).serializeToString())`